src/overlays/actors/ovl_Boss_Va/z_boss_va.c

```diff
@@ -1,53 +1,194 @@
+/*
+ * File: z_boss_va.c
+ * Overlay: ovl_Boss_Va
+ * Description: Barinade
+ */
+
 #include "z_boss_va.h"
+#include "objects/object_bv/object_bv.h"
+#include "overlays/actors/ovl_En_Boom/z_en_boom.h"
 #include "objects/gameplay_keep/gameplay_keep.h"
 
 #define FLAGS 0x00000035
 
 #define THIS ((BossVa*)thisx)
+#define BODY(this) ((BossVa*)this->actor.parent)
```

Comment:
I don't think it makes sense to keep the name `BODY` for the macro if it takes an argument. I think I suggested `GET_BODY` for this reason. Also make sure to use parens for any macro argument so we avoid classic macro issues.
```suggestion
#define GET_BODY(this) ((BossVa*)(this)->actor.parent)
```